src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="container h-75"
+     *ngIf="!startClusterCreation">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="blank-page">
+      <!-- htmllint img-req-src="false" -->
+      <img [src]="projectConstants.cephLogo"
+           alt="Ceph"
+           class="img-fluid mx-auto d-block">
+      <h3 class="text-center m-2"
+          i18n>Welcome to {{ projectConstants.projectName }}</h3>
+
+      <div class="m-4">
+        <h4 class="text-center"
+            i18n>Please expand your cluster first</h4>
+        <div class="offset-md-2">
+          <button class="btn btn-accent m-2"
+                  name="expand-cluster"
+                  (click)="createCluster()"
+                  i18n>Expand Cluster</button>
+          <button class="btn btn-light"
+                  name="skip-cluster-creation"
+                  (click)="skipClusterCreation()"
+                  i18n>Skip</button>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="card"
+     *ngIf="startClusterCreation">
+  <div class="card-header"
+       i18n>Expand Cluster</div>
+  <div class="container-fluid">
+    <cd-wizard [stepsTitle]="stepTitles"></cd-wizard>
+    <div class="card-body vertical-line">
+      <ng-container [ngSwitch]="currentStep?.stepIndex">
+        <div *ngSwitchCase="'1'"
+             class="ml-5">
+          <h4 class="title"
+              i18n>Add Hosts</h4>
+          <br>
+          <cd-hosts [clusterCreation]="true"></cd-hosts>
```

Comment:
Add a attribute to hide the `Add Hosts` title in this component.
![Bildschirmfoto vom 2021-08-09 16-11-33](https://user-images.githubusercontent.com/1897962/128720522-3ff66568-6f2e-476b-8e74-1b0a521d87de.png)

Reply:
Personally, I feel its meaningful to show that title on the top. Otherwise that area will be just a table where user can add the hosts. Its a little bit more convenient to show the title on the top of that table and TBH there's not a lot of repetition happening there.